pysph/sph/tests/test_gjsolve.py

```diff
@@ -0,0 +1,23 @@
+from pysph.sph.wc.density_correction import gj_Solve
+import numpy as np
+import unittest
+
+
+class TestGjSolve(unittest.TestCase):
+
+    def test_gj_solve(self):
+        for i in range(10):
```

Comment:
Random matrices make it hard to test and debug as the matrices change each time.  Please use a more deterministic test.  Use specific matrices, also pick bad matrices and see what it does.